package-lock.json

```diff
@@ -2475,7 +2475,6 @@
         "jest-resolve": "^26.6.2",
         "jest-util": "^26.6.2",
         "jest-worker": "^26.6.2",
-        "node-notifier": "^8.0.0",
```

Comment:
Are these changes needed?

Reply:
nope. not intentional. I'll get rid of them